src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -309,15 +311,16 @@
          {:endpoint (endpoints/create-template method-ref)
           :payload method-ref
           :headers utils/content-type=json
-          :on-done (fn [{:keys [success? get-parsed-response status-text]}]
-                     (if success?
-                       (let [response (get-parsed-response false)]
+          :on-done (fn [{:keys [success? get-parsed-response]}]
+                     (let [response (get-parsed-response false)]
+                       (if success?
                          (endpoints/call-ajax-orch
                            {:endpoint endpoints/get-inputs-outputs
                             :payload (response "methodRepoMethod")
                             :headers utils/content-type=json
                             :on-done (fn [{:keys [success? get-parsed-response]}]
                                        (swap! state dissoc :blocker)
+                                       (swap! state dissoc :parse-error)
```

Comment:
This will result in two re-renders.  Combine:

`(swap! state dissoc :blocker :parse-error)`